src/Provider/Fetcher.js

```diff
@@ -15,6 +15,25 @@ const arrayBuffer = (url, options = {}) => fetch(url, options).then((response) =
     return response.arrayBuffer();
 });
 
+const mapbox = (url, options = {}) => {
```

Comment:
This fethchershould be more generic, as 'mutlisource' and then know it needs to call the arraybuffer fetcher several time. (or an other one if needed.

Maybe it should be the fetcher arrayBuffer (and other) that are able to deal with multisource.